editoast/src/views/v2/timetable.rs

```diff
@@ -151,95 +132,22 @@ async fn get(
     Ok(Json(timetable.into()))
 }
 
-#[derive(Serialize, ToSchema)]
-struct ListTimetablesResponse {
-    #[serde(flatten)]
-    stats: PaginationStats,
-    results: Vec<TimetableResult>,
-}
-
-/// Retrieve paginated timetables
-#[utoipa::path(
-    get, path = "",
-    tag = "timetablev2",
-    params(PaginationQueryParam),
-    responses(
-        (status = 200, description = "List timetables", body = inline(ListTimetablesResponse)),
-    ),
-)]
-async fn list(
-    db_pool: State<DbConnectionPoolV2>,
-    Query(pagination_params): Query<PaginationQueryParam>,
-) -> Result<Json<ListTimetablesResponse>> {
-    let settings = pagination_params
-        .validate(1000)?
-        .warn_page_size(100)
-        .into_selection_settings();
-    let (timetables, stats) =
-        Timetable::list_paginated(db_pool.get().await?.deref_mut(), settings).await?;
-    Ok(Json(ListTimetablesResponse {
-        stats,
-        results: timetables.into_iter().map_into().collect(),
-    }))
-}
-
-/// Create a timetable
```

Comment:
Nit: can we keep this code comment here? Without this the OpenAPI endpoint is missing a description.